static/app/views/issueDetails/quickTrace/issueQuickTrace.tsx

```diff
@@ -1,89 +1,110 @@
-}: Props) {
-  const {shouldShowPrompt, snoozePrompt} = usePromptCheck({
-    projectId: event.projectID,
-    feature: 'quick_trace_missing',
-    organization,
-  });
+function PlaceholderEventNode({group}: {group: Group}) {
+  if (group.issueCategory !== IssueCategory.ERROR) {
+    return (
+      <EventNode type="black" icon={null}>
+        {t('This Event')}
+      </EventNode>
+    );
+  }
 
-  if (
-    !quickTrace ||
-    quickTrace.error ||
-    quickTrace.trace === null ||
-    quickTrace.trace.length === 0
-  ) {
-    if (!shouldShowPrompt) {
-      return null;
-    }
+  return (
+    <EventNode type="error" data-test-id="event-node">
+      <ErrorNodeContent>
+        <IconFire size="xs" />
+        {t('This Event')}
+      </ErrorNodeContent>
+    </EventNode>
+  );
+}
 
+function TransactionMissingPlaceholder({
+  type,
+  group,
+}: {
+  group: Group;
+  type?: QuickTraceQueryChildrenProps['type'];
+}) {
+  const organization = useOrganization();
+  useEffect(() => {
     trackAdvancedAnalyticsEvent('issue.quick_trace_status', {
       organization,
-      status: quickTrace?.type === 'missing' ? 'transaction missing' : 'trace missing',
-      is_performance_issue: isPerformanceIssue ?? false,
+      status: type === 'missing' ? 'transaction missing' : 'trace missing',
+      is_performance_issue: group.issueCategory === IssueCategory.PERFORMANCE,
     });
+  });
 
-    return (
-      <StyledAlert
-        type="info"
-        showIcon
-        trailingItems={
-          <Button
-            aria-label={t('Remind me later')}
-            title={t('Dismiss for a month')}
-            priority="link"
-            size="xs"
-            icon={<IconClose />}
-            onClick={snoozePrompt}
-          />
-        }
-      >
-        {tct(
-          'The [type] for this event cannot be found. [link:Read the docs to understand why].',
+  return (
+    <QuickTraceWrapper>
+      <Tooltip
+        isHoverable
+        position="bottom"
+        title={tct(
+          'The [type] for this event cannot be found. [link:Read the  docs] to understand why.',
           {
-            type: quickTrace?.type === 'missing' ? t('transaction') : t('trace'),
+            type: type === 'missing' ? t('transaction') : t('trace'),
             link: (
               <ExternalLink href="https://docs.sentry.io/product/sentry-basics/tracing/trace-view/#troubleshooting" />
             ),
           }
         )}
-      </StyledAlert>
-    );
+      >
+        <QuickTraceContainer data-test-id="missing-trace-placeholder">
+          <EventNode type="white" icon={null}>
+            ???
+          </EventNode>
+          <TraceConnector />
+          <PlaceholderEventNode group={group} />
+        </QuickTraceContainer>
+      </Tooltip>
+    </QuickTraceWrapper>
+  );
+}
+
+function IssueQuickTrace({group, event, location, organization, quickTrace}: Props) {
+  if (
+    !quickTrace ||
+    quickTrace.error ||
+    quickTrace.trace === null ||
```

Comment:
@malwilley what about `quickTrace.trace === undefined`? We aren't always good about null vs undefined with serialized data.

Reply:
This was the existing logic so it should work fine, but I agree it should be using the `defined` function instead